src/HasEnums.php

```diff
@@ -28,6 +30,86 @@ public function getAttribute($key)
             : $value;
     }
 
+    /**
```

Comment:
These doc-tags are only for IDEs - and the `@see` annotation for faster debugging.